src/lib/mongoBackend/mongoQueryContext.cpp

```diff
@@ -38,6 +38,268 @@
 
 /* ****************************************************************************
 *
+* someContextElementNotFound -
+*
+* Returns true if some attribut with 'found' set to 'false' is found in the CER vector passed
+* as argument
+*
+*/
+bool someContextElementNotFound(ContextElementResponseVector& cerV)
+{
+  for (unsigned int ix = 0; ix < cerV.size(); ++ix)
+  {
+    for (unsigned int jx = 0; jx < cerV.get(ix)->contextElement.contextAttributeVector.size(); ++jx)
+    {
+      if (!cerV.get(ix)->contextElement.contextAttributeVector.get(jx)->found)
+      {
+        return true;
+      }
+    }
+  }
+  return false;
+}
+
+
+/* ****************************************************************************
+*
+* searchCprForAttribute -
+*
+* Search the CPr, given the entity/attribute as argument. Actually, two CPrs can be returned
```

Comment:
The parameters is a C++ reference, thus it is valid for returning things. If the problem is the function preamble comment, maybe you could suggest a better wording, please?